legend-engine-language-pure-compiler/src/main/java/org/finos/legend/engine/language/pure/compiler/toPureGraph/PureModel.java

```diff
@@ -469,44 +456,11 @@ private void loadDataElements(PureModelContextDataIndex pure)
         pure.dataElements.forEach(this::processSecondPass);
     }
 
-    private void loadStores(PureModelContextDataIndex pure)
-    {
-        this.extensions.sortExtraProcessors(pure.stores.keysView()).forEach(p ->
-        {
-            MutableList<org.finos.legend.engine.protocol.pure.v1.model.packageableElement.store.Store> stores = pure.stores.get(p);
-            stores.forEach(this::processSecondPass);
-            stores.forEach(this::processThirdPass);
-            stores.forEach(this::processFourthPass);
-            stores.forEach(this::processFifthPass);
-        });
-    }
-
-    public void loadMappings(PureModelContextDataIndex pure)
-    {
-        pure.mappings.forEach(this::processSecondPass);
-        pure.mappings.forEach(this::processThirdPass);
-        pure.mappings.forEach(this::processFourthPass);
-        pure.mappings.forEach(this::processFifthPass);
-
-    }
-
-    public void loadConnectionsAndRuntimes(PureModelContextDataIndex pure)
-    {
-        // Connections must be loaded before runtimes
-        pure.connections.forEach(this::processSecondPass);
-        pure.runtimes.forEach(this::processSecondPass);
-    }
-
     private void loadOtherElementsPreStores(PureModelContextDataIndex pure)
```

Comment:
I think this kind of changes should go in its own PR so we can easily rollback in case. You would also want to remote run this with the internal codebase where we have more plugins

Reply:
Also, I'm not even sure if this is the right thing to do generally. This means somebody who's going to add a new DSL need to know about the implementation of the processing passes of a lot of other plugins. Is that really what we want?

Reply:
Agree this is something very sensitive, and we might want to change on a separate PR before we make further changes that depend on this change.

As for the knowledge people need to have - I think that is true regardless of this implementation.  What we don't have is clear documentation into what should go on each phase and what the core part of the platform does on each phase.  Like, if I create an element that depends on Store, I need to know what I should do on each compilation phase  of my component, specially if it needs Store to be on specific state.

Reply:
Solved this without re-ordering compilation